tutorials/vue/vue-websocket.md

```diff
@@ -0,0 +1,45 @@
+[WebSockets](/tutorials/node/websockets) are a great tool for when you want to show real time changes in data.
+For example, a server can push stock market price changes to the client rather than the client needing to ask for the changes via a http request, or in chat applications which reuse the WebSocket connection for the current users.
```

Comment:
Capitalize HTTP and remove the chat application example. Readers likely are already aware of why they would want to use a websocket, they want to know how, so skip to the good part :)